packages/cli/assets/pluginPackage/src/modelDefine.ts

```diff
@@ -0,0 +1,23 @@
+import {
+  ModelDefineType,
+  UniDataset,
+  ModelDefineArgsType,
+  UniModel,
+  Sample
+} from '@pipcook/pipcook-core';
+
+const templateModelDefine: ModelDefineType = async (data: UniDataset, args: ModelDefineArgsType): Promise <UniModel> => {
+
+  const model = {}; // Some model defination
+
+  const retModelDefine: UniModel = {
+    model,
+    predict: async function (input: Sample) {
+      const pred = this.model.predict(input.data);
+      return pred.toString();
+    }
+  };
+  return retModelDefine;
```

Comment:
Not a Promise object.

Reply:
Because of the outer `async`(line 9), this object is indeed a Promise Object.